packages/storybook/src/nimble/spinner/spinner.mdx

```diff
@@ -13,9 +14,15 @@ import { spinnerTag } from '../../../../nimble-components/src/spinner';
     accent).
```

Comment:
I think we should remove the line that begins with "It has 3 sizes..." The API section below is already documenting exactly that.